src/id/doujindesu/src/eu/kanade/tachiyomi/extension/id/doujindesu/DoujinDesu.kt

```diff
@@ -136,7 +136,7 @@ class DoujinDesu : ParsedHttpSource(), ConfigurableSource {
         Genre("Cunnilingus"),
         Genre("Dark Skin"),
         Genre("Daughter"),
-        Genre("Defloartion"),
+        Genre("Defloration"),
```

Comment:
Ohh i just realize that (⁠・⁠–⁠・⁠;⁠)⁠ゞ
Sorry for the trouble I've caused
I didn't think that I had a typo in writing the genre, because I never use that genre
Well, the writing was still manual and didn't use a script to extract the genre, that was my mistake at that time